### PR TITLE
Remove Jaeger from local cluster as it is mostly unused + editorconfig fix

### PR DESCRIPTION
**Pull request by jakubfijalkowski**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -13,6 +13,9 @@ indent_size = 2
 [*.{xml, targets, csproj, props}]
 indent_size = 2
 
+[*.tf]
+indent_size = 2
```
Comment (Saancreed) [inline .editorconfig:17]: Sneaking in unrelated change? :police_car:

**Review by Saancreed — APPROVED** [review]
Sure I guess :shrug:

File: .editorconfig (hunk 1 above)
Comment (jakubfijalkowski) [reply .editorconfig:17]: Indeed I am.
